locales/es.json

```diff
@@ -14,42 +14,42 @@
 	"passenger-request-destination.provide_destination": "¿A donde desea ir? Por favor escriba la dirección exacta de la calle (Así el sistema GPS podrá entender) o describa el lugar.",
 	"order-submitted.order_submitted": "⏳ Localizando conductores... Si usted encontro un conductor, presione el botón de abajo 👇",
 	"order-submitted.cancel": "Terminar búsqueda",
-	"order-submitted.error_incorrect_input": "Opción desconocida, por favor intente de nuevo.",
+	"order-submitted.error_incorrect_input": "Opción desconocida, por favor intentelo de nuevo.",
```

Comment:
Guys maybe we should decide if the translation should be usted or tu. Warhammer do you think @jbrizio?